client/my-sites/domains/components/domain-warnings/index.jsx

```diff
@@ -420,6 +424,72 @@ export class DomainWarnings extends React.PureComponent {
 		);
 	};
 
+	newTransfersWrongNS = () => {
+		const newTransfers = this.getDomains().filter(
+			domain =>
+				domain.registrationMoment &&
+				moment( domain.registrationMoment )
+					.add( 3, 'days' )
+					.isAfter( moment() ) &&
+				domain.transferStatus === transferStatus.COMPLETED &&
+				! domain.hasWpcomNameservers
+		);
+
+		if ( newTransfers.length === 0 ) {
+			return null;
+		}
+
+		const { translate } = this.props;
+		const compactMessage = translate( 'Update nameservers' );
```

Comment:
How about something a bit less jargony? 😁 `nameservers` might not mean anything to the user. Something more generic like `Domain needs updating` (`Domain requires action`?) might captivate them more.

Reply:
Yep...I like it.